apis/profileAPI.ts

```diff
@@ -35,3 +35,13 @@ export const changePasswordRequest = async (userId: string | null, data: ChangeP
   });
   if (!res.ok) throw new Error("Kunne ikke opdatere adgangskode");
 };
+
+export const deleteUserRequest = async (userId: string | null) => {
+  if (userId === null) {
```

Comment:
Why do the deleteIserRequest receive "null" and right after you ensure that userId isnt null. Shouldnt you be able to do deleteUserRequest = async (userId:string) => {}?